src/panel/poi/blocks/Phone.jsx

```diff
@@ -1,16 +1,65 @@
-/* globals _ */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-
 import Block from 'src/panel/poi/blocks/Block';
+import { isFromPagesJaunes } from 'src/libs/pois';
+import { useDevice, useI18n } from 'src/hooks';
+import { IconFileCopy } from 'src/components/ui/icons';
+import { copyToClipboard } from 'src/libs/clipboard';
+import Telemetry from 'src/libs/telemetry';
+
+const PhoneBlock = ({ block, poi }) => {
+  const { _ } = useI18n();
+  const { isMobile } = useDevice();
+  const [isHidden, setHidden] = useState(!isMobile && isFromPagesJaunes(poi));
+
+  useEffect(() => {
+    setHidden(!isMobile && isFromPagesJaunes(poi));
+  }, [poi, isMobile]);
+
+  const sendEvent = () => {
+    Telemetry.sendPoiEvent(
+      poi,
+      'phone',
+      Telemetry.buildInteractionData({
+        id: poi.id,
+        source: poi.meta?.source,
+        template: 'single',
+        zone: 'detail',
+        element: 'phone',
+      })
+    );
+  };
+
+  const mobileProps = {
+    href: block.url,
+    onClick: sendEvent,
+  };
+
+  const desktopProps = isHidden
+    ? {
+        onClick: () => {
+          setHidden(false);
+          sendEvent();
+        },
+      }
+    : {
+        onClick: () => copyToClipboard(block.local_format),
+        actionIcon: <IconFileCopy width={20} />,
+        tooltip: _('Copy'),
```

Comment:
I am not sure about this change. This action was already doable manually, whereas the standard "href" link is now lost.   
On most systems, it can be used to forward the number to a suitable application, using the international format that is now completely inaccessible.

Reply:
how about a "copy" button + a "call" button aside to the number?

Reply:
The designer's idea is that every "block" should have a **unique** and **contextual** action which is the more useful and logical for the user on the current device.
For desktop it's been changed to "copy to clipboard". It may seem like we lost something, but actually there is already a "call" CTA clearly visible in the button bar, and phone/desktop integration is not that common, so I see the point.

Reply:
Ok, it was decided "IRL" to discuss this copy-to-clipboard action later separately, and keep the block as a link to the `tel:` url even on desktop for now, so I reverted some of the latest commits.